platform/representation/src/MCTRepresentation.js

```diff
@@ -159,10 +162,11 @@ define(
                         path = representation && getPath(representation),
                         uses = ((representation || {}).uses || []),
                         canRepresent = !!(path && domainObject),
+                        canEdit = !!(domainObject && domainObject.hasCapability('editor')),
                         id = domainObject && domainObject.getId(),
                         key = $scope.key;
 
-                    if (unchanged(canRepresent, id, key)) {
+                    if (unchanged(canRepresent, canEdit, id, key)) {
```

Comment:
Had to modify this slightly. Need the representation to refresh its capabilities and actions when it becomes (un)editable.